sources/commands/version.ts

```diff
@@ -56,9 +56,13 @@ export default class Version extends BaseCommand {
       "release-tool version",
       this.context
     );
-    const { lastTagName, lastVersion } = await git.getLastTag();
+    const { lastTagName, lastVersion } = await git.getLastTag(
+      this.version === "prerelease"
+    );
```

Comment:
What will happen when we release `8.0.0-beta.0`, by running `yarn release-tool version --version 8.0.0-beta.0`?

Reply:
Yes. And then `yarn release-tool version prerelease` should bump `.0` to `.1` and so on. If we want to publish Babel 7 after `beta.0` is published, we might need extra works such as writing to the `version_babel8` field instead of `version` field (I don't know if we already handle that)